reference-architectures/cloud_deploy_flow/CloudBuild/buildCloudRun.yaml

```diff
@@ -0,0 +1,20 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  dir: 'CloudRun/'
+  args: ['build', '-f', 'dockerfile', '-t', 'us-central1-docker.pkg.dev/$PROJECT_ID/random-date-app/app:latest', '.']
```

Comment:
Nit/opinion

The default for `-f` is `Dockerfile` (https://docs.docker.com/reference/cli/docker/buildx/build/#file)

If you rename `dockerfile` to `Dockerfile` you can simplify this command by removing the `-f` option entirely.

Reply:
Got it! I did not know this, thank you!